### ISSUE TITLE
I really think the screen reader version of trimps should have sounds.

### ISSUE THREAD

**Issue by audiogamer22** [Trimps/Trimps.github.io#312]
I'm blind and use NVDA with firefox and am enjoying trimps but I really feel like sounds could help us to understand certain things, for example; how full the  trimps breeding bar is, when a trimp or trimps have killed an enimie, and more, but make it toggleable, it would really help us to have these sounds instead of having to keep read through the page, it's Icons but for your ears.

also the maps tutorial mentions a button, I think, that allows you to run a map for certain things like upgrades, I don't think that button is visible for screen reader users.

**Comment by Keithcat1** [comment]
Why are you looking at breed  so often? Adding more than one or two sounds won't help because ether you toggle them on and off all the time or they all play at once and you have trouble hearing which is which. Also total breed time is in the game overview.

**Comment by audiogamer22** [comment]
I just used it as an example, I actually think that I'd just prefer a sound for when a trimp kills something, trimps die, and trimps get upgrades/ and perhaps helium, but i'm not sure on that one as i've not got there yet.

**Comment by audiogamer22** [comment]
Closed as I don't play the game anymore and if peple want sounds i'm sure they'll open another issue.
